build-azure.json

```diff
@@ -16,7 +16,7 @@
          "tenant_id": "{{user `arm_tenant_id`}}",
 
          "location": "West Europe",
-         "vm_size": "Standard_A1",
+         "vm_size": "Standard_D2_v2",
```

Comment:
Main reason behind this upgrade is about saving some build time (up to 8 minutes less per build). Not sure in terms of costs, but in principle we should be even out as yielding shorter build process, despite using a more powerful machine.